appinfo/routes.php

```diff
@@ -200,31 +200,42 @@
 			'url' => '/api/{apiVersion}/question/reorder',
 			'verb' => 'POST',
 			'requirements' => [
-				'apiVersion' => 'v2(\.[1-4])?'
+				'apiVersion' => 'v2(\.[1-5])?'
 			]
 		],
 		[
 			'name' => 'api#reorderQuestions',
 			'url' => '/api/{apiVersion}/question/reorder',
 			'verb' => 'PUT',
 			'requirements' => [
-				'apiVersion' => 'v2\.[2-4]'
+				'apiVersion' => 'v2\.[2-5]'
 			]
 		],
 		[
 			'name' => 'api#deleteQuestion',
 			'url' => '/api/{apiVersion}/question/{id}',
 			'verb' => 'DELETE',
 			'requirements' => [
-				'apiVersion' => 'v2(\.[1-4])?'
+				'apiVersion' => 'v2(\.[1-5])?'
 			]
 		],
 		[
 			'name' => 'api#cloneQuestion',
 			'url' => '/api/{apiVersion}/question/clone/{id}',
 			'verb' => 'POST',
 			'requirements' => [
-				'apiVersion' => 'v2\.[3-4]'
+				'apiVersion' => 'v2\.[3-5]'
+			]
+		],
+
+		// Batch processing for reordering of options
+		[
+			'name' => 'api#reorderOptions',
+			'url' => '/api/{apiVersion}/question/{id}/options',
+			'verb' => 'PATCH',
+			'requirements' => [
+				'id' => '\d+',
+				'apiVersion' => 'v2(\.5)?'
```

Comment:
I'd switch those two lines. Makes more sense alphabetically and also based on the appearance in the URL
```suggestion
				'apiVersion' => 'v2(\.5)?',
				'id' => '\d+'
```